xwiki-commons-core/xwiki-commons-legacy/xwiki-commons-legacy-velocity/src/main/java/org/xwiki/velocity/VelocityFactory.java

```diff
@@ -30,8 +30,10 @@
  * we need this in XWiki since we want to allow each Skin to provide its own global macros.
  *
  * @version $Id$
+ * @deprecated use {@link VelocityManager} instead
  */
 @Role
+@Deprecated(since = "15.9-rc-1")
```

Comment:
Don't we use the `15.9RC1` version format here? At least that's what I see when I look for other similar code.